src/renderer/views/Channel/Channel.vue

```diff
@@ -3,7 +3,7 @@
     ref="search"
   >
     <ft-loader
-      v-if="isLoading"
+      v-if="isLoading && !isErrorMessage"
```

Comment:
I can't find the `isErrorMessage` anywhere else in the code, I'm assuming that you meant `errorMessage` here?
```suggestion
      v-if="isLoading && !errorMessage"
```

Reply:
Good catch